cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java

```diff
@@ -100,6 +101,18 @@ public static void pushPackages(String packageName, String sourceRoot, String in
         }
 
         String orgName = manifest.getName();
+        // Validate the org-name
+        if (!validateOrg(orgName)) {
+            throw createLauncherException("invalid organization name provided \'" + orgName + "\'." + " Only " +
+                                          "lowercase alphanumerics and underscores are allowed in an organization " +
+                                          "name and the maximum length is 256 characters");
+        }
+        // Validate the package-name
+        if (!validatePkg(packageName)) {
+            throw createLauncherException("invalid package name provided'" + packageName + "\'." + " Only " +
```

Comment:
We can merge `"\'." + " Only "` as well.

Reply:
Merged both with af18af2